packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -53,7 +56,7 @@ export const PaymentConfirmation = ({
   });
 
   const planPrice = formatPlanPricing(
-    amount,
+    coupon && amount ? amount - coupon.amount : amount,
```

Comment:
Maybe capture this into a variable?  Line 139 below is doing the same thing.

Reply:
done